src/extension/content-script/batteries/Medium.ts

```diff
@@ -0,0 +1,74 @@
+import getOriginData from "../originData";
+import setLightningData from "../setLightningData";
+
+const urlMatcher = /^https:\/\/(.+\.)?medium.com\/(\S+)?/;
+
+const battery = (): void => {
+  const authorLinkSelector = getAuthorLinkSelector();
+  if (!authorLinkSelector) return;
+
+  const shortBio = document.querySelector<HTMLParagraphElement>(
+    `${authorLinkSelector}~p:last-of-type`
+  )?.innerText;
+
+  if (shortBio) {
+    const match = shortBio.match(/(⚡:?)\s?(\S+@\S+)/i);
```

Comment:
I think we should use the same regex as in the others and also allow lnurlp: or lightning: there. 

https://github.com/getAlby/lightning-browser-extension/blob/master/src/extension/content-script/batteries/YouTubeVideo.ts#L25

what do you think? 
ideally we have one place where we define that regex